packages/appkit/src/client.ts

```diff
@@ -901,7 +901,8 @@ export class AppKit {
   }
 
   private extendDefaultCaipNetwork(options: AppKitOptions) {
-    const defaultNetwork = options.networks.find(n => n.id === options.defaultNetwork?.id)
+    const defaultNetwork =
+      options.defaultNetwork || options.networks.find(n => n.id === options.defaultNetwork?.id)
```

Comment:
Does this line make sense? If first part is undefined, secondpart will never get an id?

Reply:
Great point, the defaultNetwork should have been used only in `getDefaultNetwork` function